src/debugger/debug_workflow.ts

```diff
@@ -202,6 +214,32 @@ export class WorkflowContextDebug extends DBOSContextImpl implements WorkflowCon
     return this.startChildWorkflow(wf, ...args);
   }
 
+  invokeOnConfig<T extends object>(targetCfg: ConfiguredClass<T>): InvokeFuncsConf<T> {
+    const ops = getRegisteredOperations(targetCfg.ctor);
+
+    const proxy: Record<string, unknown> = {};
+    for (const op of ops) {
+      proxy[op.name] = op.txnConfig
+        ?
+        (...args: unknown[]) => this.transaction(op.registeredFunction as Transaction<unknown[], unknown>, targetCfg, ...args)
+        : op.commConfig
+          ?
+          (...args: unknown[]) => this.external(op.registeredFunction as Communicator<unknown[], unknown>, targetCfg, ...args)
+          : undefined;
+    }
+    return proxy as InvokeFuncsConf<T>;
+  }
+
+  async startChildWorkflowOnConfig<T extends unknown[], R>(targetCfg: ConfiguredClass<unknown>, wf: Workflow<T, R>, ...args: T): Promise<WorkflowHandle<R>> {
```

Comment:
Can we keep the name `startChildWorkflow` the same?

```suggestion
  async startChildWorkflow<T extends unknown[], R>(targetCfg: ConfiguredClass<unknown>, wf: Workflow<T, R>, ...args: T): Promise<WorkflowHandle<R>> {
```

Reply:
Maybe.  I am not happy with startChildWorkflow in the first place, still thinking about what I can do here.  Not this direct suggestion though because then we have two implementations of the function.

Reply:
So I was a bit on the fence about whether it ought to be, for example:
```ctx.startChildWorkflow(cfc, S3Ops.deleteFile, fileDetails);```
or
```ctx.startChildWorkflow(cfc, 'deleteFile', fileDetails);```

The former is more consistent with our existing syntax; the latter may have been a better syntax in the first place.  I went with the former.

Reply:
(The checking is pretty good.)